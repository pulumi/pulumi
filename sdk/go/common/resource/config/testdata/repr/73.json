{"my:key":{"bar":"baz","secure":"value"}}