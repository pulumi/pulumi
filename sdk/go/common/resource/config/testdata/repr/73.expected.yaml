decrypt:
  my:key: '{"bar":"baz","secure":"value"}'
paths:
  my:key:
    value:
      bar: baz
      secure: value
    string: '{"bar":"baz","secure":"value"}'
    redacted: '{"bar":"baz","secure":"value"}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAAAu/4IrAAIDYmFyBnN0cmluZwwFAANiYXoGc2VjdXJlBnN0cmluZwwHAAV2YWx1ZQ==
    secure: false
    isObject: true
  my:key.bar:
    value: baz
    string: baz
    redacted: baz
    object: EBAABnN0cmluZwwFAANiYXo=
    secure: false
    isObject: false
  my:key.secure:
    value: value
    string: value
    redacted: value
    object: EhAABnN0cmluZwwHAAV2YWx1ZQ==
    secure: false
    isObject: false
